.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+tab_width = 4
```

Comment:
Should tab width be 2?

Reply:
All other projects have tabs with 4.
E.g.
https://github.com/aion-dk/dbb/blob/main/.editorconfig